foo/config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "flavour": "foo/flavour.json", //throw an error if you can't find that file
```

Comment:
JSON files cannot have comments. If you intend to put some, you might achieve this by adding explicitly named fields, such as `flavour_comment`